zng/docs/spec.md

```diff
@@ -511,15 +538,16 @@ An application-specific payload has the following form:
 Here, `<control code>` is a decimal integer in the range 6-127 and `<payload>`
```

Comment:
Suggestion: Replace "Here, `<control code>`" with "`Where, `<control code>`"

Reply:
Actually, if I made that change, I'd think we should drop the comma as well. @nwt is usually good for a tiebreaker in nit situations. Any opinion @nwt?

Reply:
"Where A is B and C is D." doesn't make sense to me.

Reply:
Oops I meant without comma.

Reply:
@nwt: Is that your way of saying you prefer the original wording?

Reply:
@philrz: It is, but only because the suggested change appears to turn the sentence into a sentence fragment.

Reply:
@nwt: Agreed. Based on that I think I'll decline @henridf's suggestion here. Thanks to both for weighing in. 😄